hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/commit/terasort/ITestTerasortOnS3A.java

```diff
@@ -92,13 +118,27 @@ public boolean useInconsistentClient() {
     return false;
   }
 
+  @Override
+  protected String committerName() {
+    return committerName;
+  }
+
   @Override
   public void setup() throws Exception {
     super.setup();
     requireScaleTestsEnabled();
     prepareToTerasort();
```

Comment:
In prepareToTerasort - the config is modified via getYarn().getConfig.set() ... 
This is going to change the config for every test that makes use of this cluster. Likely better to create a new Configuration instance for the Job being submitted, and setting these config parameters at the job level.

Reply:
good point -I'll clone